tests/testCountInputs.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2008-2015 LSST/AURA
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+"""
+Tests for CountInputs measurement algorithm
+"""
+
+import unittest
+import lsst.utils.tests as utilsTests
+
+import lsst.pex.config as pexConfig
+import lsst.afw.geom as afwGeom
+import lsst.afw.table as afwTable
+import lsst.afw.image as afwImage
+import lsst.afw.coord as afwCoord
+import lsst.afw.detection as afwDetection
+import lsst.meas.base as measBase
+from lsst.afw.geom.polygon import Polygon
+
+def ccdVennDiagram(exp, showImage=True, legendLocation='best'):
+    '''
+    Create a figure with the bounding boxes for each of the images which go into a coadd,
+    over-plotting the given exposure object.
+    exp [in] - (Exposure) The exposure object to plot, must be the product of a coadd
+    Optional:
+    showImage [in] - (Bool) Plot the data in exp in addition to just it's bounding box, default True
+    LegendLocation [in] - (String) Matplotlib legend location code, can be: 'best', 'upper right',
+                             'upper left', 'lower left', 'lower right', 'right', center left',
+                             'center right', 'lower center', 'upper center', 'center'
+    '''
+    # Create the figure object
+    fig = plt.figure()
```

Comment:
This fails (`NameError: global name 'plt' is not defined`). Simply importing it inside the calling function isn't adequate.